orca-remote-stage/src/main/kotlin/com/netflix/spinnaker/config/CallbackConfigurationProperties.kt

```diff
@@ -0,0 +1,19 @@
+package com.netflix.spinnaker.config
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+
+@ConfigurationProperties("remote-stage")
+class CallbackConfigurationProperties {
+  var http: HttpConfigurationProperties = HttpConfigurationProperties()
+  var pubsub: PubsubConfigurationProperties = PubsubConfigurationProperties()
+
+  inner class HttpConfigurationProperties {
+    var url: String = "configure-me"
+  }
+
+  inner class PubsubConfigurationProperties {
+    var id: String = "configure-me"
+    var provider: String = "configure-me"
+    var providerConfig: Map<String, String> = mapOf()
+  }
```

Comment:
Oh, yea I am actually planning on removing the pubsub callback since I think we will need some message validation and authz prior to hitting Orca and I think doing that in Gate is the best option for now.